packages/@aws-cdk/core/test/test.app.ts

```diff
@@ -301,6 +301,31 @@ export = {
     test.done();
   },
 
+  'version reporting includes @aws-solutions-konstruk libraries'(test: Test) {
+    const response = withApp({ runtimeInfo: true }, app => {
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      const resolved = require('path').resolve('test/@aws-solutions-konstruk/foo');
```

Comment:
Please move this from `test/@aws-solutions-konstruk/foo` to simply `test/runtime-info-konstruk-fixture` (the directory name does not have to be the same as the module name).